packages/tideline/css/tideline-colors.less

```diff
@@ -71,6 +71,10 @@
 // physical activty colors
 @physicalactivity: #00D3E6;
 
+// Zen mode Event colors
+@zenmode: #9FCC93;
+@zentext: #85B179;
```

Comment:
I would have set the colors in https://github.com/mdblp/blip/blob/dblp/packages/viz/src/styles/colors.css
And use the var() css function to get it